src/Domain/DomainHelpers.hpp

```diff
@@ -231,16 +231,18 @@ std::vector<std::array<size_t, 8>> corners_for_biradially_layered_domains(
 /// Set the `radial_distribution` to select the radial distribution of grid
 /// points in the cylindrical shells. The innermost shell must have
 /// `domain::CoordinateMaps::Distribution::Linear` because it changes the
-/// circularity.
+/// circularity. The distribution along the z-axis for each circular
+/// disc is specified through `distribution_in_z`.
```

Comment:
circular disc -> layer

Reply:
Sorry, I must have discarded the change to this line. I was unsure, because the use of disc/disk and layer goes all throughout both Cylinder and DomainHelpers. For now I have changed most "disks" to "layers" in Cylinder, but left DomainHelpers mostly alone for now. Do you think, I should address/change it in every file I touched?

Reply:
Just do those occurrences that you come across :)